toolchains/xslt-M4/compose/metaschema-collect.xsl

```diff
@@ -66,7 +66,7 @@
         <xsl:copy>
             <xsl:attribute name="abstract" select="$metaschema-abstract-default"/>
             <xsl:copy-of select="@* except @xsi:*"/>
-            <xsl:attribute name="module" select="short-name"/>
+            <xsl:attribute name="module" select="(@module[matches(.,'\S')],short-name)[1]"/>
```

Comment:
I know we have tested the pruning code, but does this change to collect have significant implications? I will test in the OSCAL repo, but the OSCAL metaschemas make no use of `module`, but this concatenates to it will still work in the absence of a `@module` that matches the name of the `.` context, correct?

Reply:
Tested current OSCAL and it is good, so I surpress this comment in my feedback. Feel free to ignore it!